src/components/WidgetWrapper.tsx

```diff
@@ -47,6 +48,10 @@ export default function WidgetWrapper(props: PropsWithChildren<WidgetWrapperProp
       console.warn(`Widget width must be at least 300px (you set it to ${props.width}). Falling back to 300px.`)
       return 300
     }
+    if (props.width && props.width > 600) {
```

Comment:
Nest these so you don't recheck props.width:

    if (props.width) {
      if (props.width < 300) {
        ...
      } else if (...) {
        ...
      }
    }